src/locales/default.json

```diff
@@ -349,6 +349,7 @@
       "information": "The minimum amount of voting power required to create a proposal"
     },
     "allowOnlyAuthors": "Allow only authors to submit a proposal",
+    "enableAliased": "Allow voting with session keys",
```

Comment:
Actually, shall we also mention it also includes proposal creation / deletion?

Reply:
Yes we should probably generalize the text and add a tooltip for further explanation

Reply:
How about: 
![image](https://user-images.githubusercontent.com/39556343/198095352-940bfa27-db57-4c95-8619-5f68a82dacf7.png)